### PR TITLE
Added filebench Test in Avocado

### PR DESCRIPTION
**Pull request by PraveenPenguin**
FileBench Test run File System Workload

Signed-off-by: Praveen K Pandey <praveen@linux.vnet.ibm.com>

### CODE REVIEW

**Comment by PraveenPenguin** [comment]
job log:https://paste.fedoraproject.org/435956/47488234/

File: fs/filebench.py [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+                        os.path.join(self.srcdir, self._testfile))
+
+    def test(self):
+
+        cmd = '%s -f  %s  ' % (os.path.join(self.srcdir,
+                                            'filebench'), self._testfile)
+
+        out = process.system_output(cmd)
+
+        self.log.info("result:" + out)
```
Comment (apahim) [inline fs/filebench.py]: So, one more test that does not need to check whether process.system_output failed or succeeded, right?

Comment (apahim) [inline fs/filebench.py]: Well, the test will exit in error case the `process.system_output()` returns non-0 status anyway.

Comment (PraveenPenguin) [inline fs/filebench.py]: @apahim yes

**Pull request merged**